### PR TITLE
OCPBUGS-14832 - update PTP ref config

### PR DESCRIPTION
**Pull request by aireilly**
Update the PTP reference configurations to align with latest RAN ZTP verified CRs. 

Version(s):
enterprise-4.10+

**Merge reviewers: There is a support exception in place for 4.10 doc updates required by Verizon** 

Issue:
https://issues.redhat.com/browse/OCPBUGS-14832

Link to docs preview:
* https://65252--docspreview.netlify.app/openshift-enterprise/latest/networking/using-ptp#configuring-linuxptp-services-as-grandmaster-clock_using-ptp
* https://65252--docspreview.netlify.app/openshift-enterprise/latest/networking/using-ptp#configuring-linuxptp-services-as-ordinary-clock_using-ptp
* https://65252--docspreview.netlify.app/openshift-enterprise/latest/networking/using-ptp#configuring-linuxptp-services-as-boundary-clock_using-ptp

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by aireilly** [comment]
@josephdrichard PTAL

File: modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc [hunk 1]
```diff
@@ -31,133 +31,7 @@ See "Configuring the PTP fast event notifications publisher" for more informatio
 .Recommended PTP boundary clock configuration
```
Comment (dfitzmau) [inline modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc:31]: This is a legacy title, but consider changing the title to move away from a recommendation to more of an example:

See https://www.ibm.com/docs/en/ibm-style?topic=information-claims-recommendations

Screenshot from 2023-09-27 12-38-43

File: modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc [hunk 2]
```diff
@@ -31,133 +31,7 @@ See "Configuring the PTP fast event notifications publisher" for more informatio
...
 [source,yaml]
 ----
----
```
Comment (dfitzmau) [inline modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc:34]: These are complete YAML configs, so the following guidelines do not apply?

Screenshot from 2023-09-27 12-40-48

File: snippets/ptp_PtpConfigMaster.yaml [hunk 3]
```diff
@@ -0,0 +1,126 @@
+# The grandmaster profile is provided for testing only
```
Comment (dfitzmau) [inline snippets/ptp_PtpConfigMaster.yaml:1]: Why is this YAML needed in the docs repo if it's for testing-purposes only?

**Review by dfitzmau — COMMENTED** [review]
Hi @aireilly . I do not fully understand the YAML configs but I reviewed the content as best I could. I added some minor comments inline.

/remove-label peer-review-in-progress

/label peer-review-done

File: modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc (hunk 2 above)
Comment (aireilly) [reply modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc:34]: https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics

File: modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc (hunk 1 above)
Comment (aireilly) [reply modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc:31]: We'll consider this for future updates thanks.

File: snippets/ptp_PtpConfigMaster.yaml (hunk 3 above)
Comment (aireilly) [reply snippets/ptp_PtpConfigMaster.yaml:1]: The YAML is pulled directly from RAN ZTP source. I'll update the YAML to remove this line.

File: modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc (hunk 1 above)
Comment (aireilly) [reply modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc]: Actually, this change makes sense. I'll update as you suggest :)

**Comment by aireilly** [comment]
@Bonnie-Block PTAL at this for a docs QE review when you can please.

**Comment by Bonnie-Block** [comment]
lgtm

**Comment by dfitzmau** [comment]
LGTM

**Review by jldohmann — APPROVED** [review]
lgtm

**Comment by jldohmann** [comment]
@aireilly it looks like the auto cherrypicks to 4.10-4.12 failed 😞 when you manually CP, please feel free to ping me to get those merged

**Comment by aireilly** [comment]
Cool thanks :)

**Pull request merged**